src/utils/api.js

```diff
@@ -24,6 +24,10 @@ export const patchArticleVotes = (article_id, votes) => {
   return ncNewsApi.patch(`/articles/${article_id}`, { inc_votes: votes });
 };
 
-export const postComment = (article_id) => {
-  return ncNewsApi.post(`/api/articles/${article_id}/comments`);
+export const postComment = (article_id, postBody) => {
```

Comment:
Its really difficult to check this ticket as it seems to have loads of stuff to do with other tickets as well